vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -91,11 +84,19 @@ void draw_game_install_dialog(GuiState &gui, HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     if (ImGui::BeginPopupModal("Game installation succes", &game_install_confirm, ImGuiWindowFlags_AlwaysAutoResize)) {
         ImGui::PopStyleColor();
+        ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 5);
         ImGui::TextColored(GUI_COLOR_TEXT, "Succesfully installed game.\n%s [%s] %s", host.io.title_id.c_str(), host.game_title.c_str(), host.game_version.c_str());
+        ImGui::Spacing();
         ImGui::Separator();
         ImGui::Spacing();
+        ImGui::Checkbox("Delete the package from the installed app?", &delete_game_file);
+        ImGui::Spacing();
         ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 2 - 30);
         if (ImGui::Button("Ok", BUTTON_SIZE)) {
+            if (delete_game_file) {
+                fs::remove(static_cast<const fs::path>(game_path));
```

Comment:
```suggestion
                fs::remove(game_path);
```
Does it not let you do this?

If not:
```suggestion
                fs::remove(fs::path(game_path));
```